src/components/BookCard.jsx

```diff
@@ -1,54 +1,65 @@
-import React, { useEffect } from 'react';
-import { useSelector, useDispatch } from 'react-redux';
-import { deleteBook, getBooks } from '../redux/books/booksSlics';
+import React from 'react';
 
-const BookCard = () => {
-  const { books, isLoading, isError } = useSelector((state) => state.books);
-  const dispatch = useDispatch();
-
-  useEffect(() => {
-    dispatch(getBooks());
-  }, [dispatch]);
-  if (isLoading) {
-    return <div>Loading...</div>;
-  }
-
-  if (isError) {
-    return <div>Error: try again</div>;
-  }
-
-  const renderBooks = books.map((book) => (
-    <div key={book.item_id} className="bookContainer">
-      <div className="bookTitle">
-        <p>{book.category}</p>
-        <p>{book.title}</p>
-        <p>{book.author}</p>
-        <ul className="buttonsList">
-          <li><button type="button">Comments</button></li>
-          <li><button type="button" onClick={() => dispatch(deleteBook(book.item_id))}>Remove</button></li>
-          <li><button type="button">Edit</button></li>
-        </ul>
-      </div>
-      <div className="bookChart">
-        <div>Chart</div>
-        <div>
-          <p>progres</p>
-          <p>COMPLETED</p>
-        </div>
-      </div>
-      <div className="bookChapter">
-        <p>currentChapter</p>
-        <p>chapterNumber</p>
-        <button type="button">Update Progress</button>
-      </div>
-    </div>
-  ));
+const BookCard = ({ books }) => {
+  const getRandomPercentage = () => {
+    return Math.floor(Math.random() * 100) + 1;
+  };
 
   return (
     <div className="mainContainer">
-      {renderBooks}
+      {books.map((book) => (
+        <div key={book.id} className="bookContainer">
+          <div className="bookTitle">
+            <p className="category">{book.category}</p>
+            <p className="title">{book.title}</p>
+            <p className="author">{book.author}</p>
+            <ul className="buttonsList">
+              <li>
+                <button className="bookBtn" type="button">
+                  Comments
+                </button>
+              </li>
+              <li>
+                <button className="bookBtn" type="button">
+                  Remove
+                </button>
+              </li>
+              <li>
+                <button className="bookBtn" type="button">
+                  Edit
+                </button>
+              </li>
+            </ul>
+          </div>
+          <div className="progress">
+            <div className="chart">
+              <div className="ProgressBar">
+                <div className="circle-wrap">
+                  <div className="circle">
+                    <div className="mask half">
+                      <div className="fill" />
+                    </div>
+                    <div className="inside-circle" />
+                  </div>
+                </div>
+              </div>
+            </div>
+            <div className="percentages">
+              <h2>{getRandomPercentage()}%</h2>
```

Comment:
- [ ] Great job so far. Like the previous reviewer noted, remember your design should be identical to that of zeplin including layout of elements. Since you didn't have placeholders on screen, the previous reviewer was unable to check. Please, kindly align the percentage circles like in the original design.

Yours
![Screenshot (1066)_LI](https://github.com/Cisco-the-wayword/Bookstore/assets/65298048/0f145d64-a642-4555-b74c-f800b3dcf613)
original
![Screenshot (1067)](https://github.com/Cisco-the-wayword/Bookstore/assets/65298048/990725ce-a721-4454-b487-766cc7f5fd81)